thunder/__init__.py

```diff
@@ -524,7 +550,23 @@ def apply_transforms_and_build_cache_entry(cd, cs, cache_info, prologue_trc, com
             if requires_grad:
                 from thunder.transforms.autodiff import grad_transform_on_trace
 
+                _trace_structured(
```

Comment:
It would be nice to have a helper function `log_thunder_trace` (or something like that). It would be easier to read and also to use.